aspnetcore/fundamentals/environments/3.1sample/EnvironmentsSample/StartupClassConventions.cs

```diff
@@ -0,0 +1,124 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.Reflection;
+
+// Define MULTI, see Program.cs
+// MethodBase.GetCurrentMethod().Name returns ctor, so we need DeclaringType
+
+namespace ClassConventions
+{
+#region snippet
```

Comment:
There's a lot of code in this snippet, and a lot of it's the same. I was going to suggest using comments like `// Configure the Development middleware pipeline` but I guess these samples should be verifiable. Perhaps each example could just use a single dummy middleware that writes a text response?